src/pages/components/progress-bar/usage.mdx

```diff
@@ -0,0 +1,504 @@
+
+### Interactions
+
+#### Part of a page
+
+While the progress bar is processing information on part of a page, you can
+still interact with other elements on the page. When the progress bar is
+finished processing it can either remain persistant on the page for confirmation
+or automatically dismiss itself depending on the usecase.
+
+<Row>
+<Column colLg={8}>
+
+![Progress bar as a part of a page](images/progress-bar-usage-19.png)
+
+</Column>
+</Row>
+
+#### Inside a container
+
+While the progress bar is processing information that could affect multiple
+pieces of content within a container, such as a card, depending on the use case,
+the whole container should become inactive until processing has been completed
+or the specific section within the container being processed should become
+inactive until processing has been completed. The user should be able to
+interact with the rest of the container or page that is unaffected by the
+progress bar.
+
+<Row>
+<Column colLg={8}>
+
+![Progress bar inside a container](images/progress-bar-usage-20.png)
+
+</Column>
+</Row>
+
+## Modifiers
+
+#### Transitions
+
+An indeterminate bar can change to a determinate indicator (finishing from left
+to right) when enough information is gathered to end the load animation.
+
+<Row>
+<Column colLg={8}>
+
+![An indeterminate bar can change to a determinate indicator](images/progress-bar-usage-21.gif)
+
+</Column>
+</Row>
+
+## Accessibility
+
+- The loading component is implemented using the ARIA 1.1 progressbar pattern.
+  It outputs an ARIA `role="progressbar"` and uses `aria-valuemin`,
+  `aria-valuemax`, and `aria-valuenow` to convey the loaded percentage to screen
+  reader users.
+
+- If the progress bar shows the loading of a specific part or region of the
+  applicaition, set `aria-busy="true"` and `aria-describedly="[progressbar-id]"`
+  on the related element to ensure assistive technologies can correctly convey
+  this relation to the user.
+
+- Only hide the label when it is absolute clear to the user which process the
+  progress bar represents. Note that you are still required to pass an
+  appropriate label which will be ready by screen readers.
+
+## Related
+
+#### Components
+
+- [Loading spinners](/components/loading/usage)
+- [Inline loading](/components/inline-loading/usage)
+- [Progress indicator](/components/progress-indicator/usage)
+
+#### Components
+
+- [Loading: skeleton states](/patterns/loading/#skeleton-states)
+
+#### Loading, inline loading, or progress bar?
+
+- If an activity displays a loading spinners on one screen, it should not show
+  the progress bar elsewhere in the design.
+- Both progress bars and circles can show either determinate or indeterminate
+  progress. The given area should help determine if a progress bar or circle is
+  best for that space. Progress bars are preferred in vertically narrow areas
+  (tables, cards, etc.) while progress circles are ideal for full page loading
+  or in very small areas.
```

Comment:
Should this duplicate part be removed? Feels like it's nearly identical to what is written up above

Reply:
@aagonzales @laurenmrice think we talked briefly about this, saying that it's good to remind people about this. Is it too redundant?